src/vue/forms/WithdrawalForm.vue

```diff
@@ -330,13 +292,16 @@ export default {
     },
     async getFees () {
       try {
-        const fees = await Sdk.horizon.fees.get(FEE_TYPES.withdrawalFee, {
-          account: this.accountId,
-          asset: this.form.asset.code,
-          amount: this.form.amount,
+        const fees = await this.calculateFees({
+          type: FEE_TYPES.withdrawalFee,
+          assetCode: this.form.asset.code,
+          amount: this.form.amount || 0,
+          senderAccountId: this.accountId,
+          recipientAccountId: this.accountId,
```

Comment:
we cannot charge recipient fee from the withdraw operation, we should show source fees only. thus you maybe need to bring some modifications to your fees-viewer cuz there is no `CoupledFees` instance returned from the `caclulateSourceFees` function